resolver/src/system_conf/windows.rs

```diff
@@ -0,0 +1,55 @@
+use std::io;
+use std::str::FromStr;
+use std::net::SocketAddr;
+
+use ipconfig::get_adapters;
+use ipconfig::computer::{get_search_list, get_domain, is_round_robin_enabled};
+
+use trust_dns::rr::Name;
+use config::{ResolverConfig, ResolverOpts, NameServerConfig, Protocol};
+
+
+macro_rules! map_ipconfig_to_io_error {
+    ($result:expr) => ($result.map_err(|e| io::Error::new(io::ErrorKind::Other, Box::new(e))));
+}
+
+/// Returns the name servers of the computer (of all adapters)
+fn get_name_servers() -> io::Result<Vec<NameServerConfig>> {
+    let adapters = map_ipconfig_to_io_error!(get_adapters())?;
+    let name_servers = adapters
+        .iter()
+        .flat_map(|adapter| adapter.dns_servers().iter())
+        .map(|ipaddr| {
+                 NameServerConfig {
+                     socket_addr: SocketAddr::new(*ipaddr, 53),
+                     protocol: Protocol::Udp,
```

Comment:
We should add both UDP and TCP here... TCP is necessary for any truncated queries.

Reply:
Done